src/definitions/behaviors/api.js

```diff
@@ -19,30 +20,30 @@
         return typeof obj === 'function' && typeof obj.nodeType !== 'number';
     }
 
-    window = (typeof window != 'undefined' && window.Math == Math)
+    window = window !== undefined && window.Math === Math
         ? window
         : globalThis;
 
-    $.api = $.fn.api = function (parameters) {
```

Comment:
Dont remove the short cut call without an attached element. It is referenced later at settings, used inside tab.js and would be a breaking change